src/database/models.py

```diff
@@ -1900,11 +1891,110 @@ def full_json(self):
+    properties_services: str
+      Whether this vendor services Residential or Commercial units only
+    onboarding_date: DateTime
+      When this vendor was onboard-ed on the MassEnergize Platform for this
+        community
+    onboarding_contact:
+      Which MassEnergize Staff/User onboard-ed this vendor
+    verification_checklist:
+      contains information about some steps and checks needed for due diligence
+      to be done on this vendor eg. Vendor MOU, Reesearch
+    is_verified: boolean
+      When the checklist items are all done and verified then set this as True
+      to confirm this vendor
+    more_info: JSON
+      any another dynamic information we would like to store about this Service
+    created_at: DateTime
+      The date and time that this Vendor was added
+    created_at: DateTime
+      The date and time of the last time any updates were made to the information
+      about this Vendor
+    is_approved: boolean
+      after the community admin reviews this, can check the box
+
+    """
+
+    service_area = models.CharField(max_length=SHORT_STR_LEN)
+    service_area_states = models.JSONField(blank=True, null=True)
+    services = models.ManyToManyField(Service, blank=True)
+    properties_serviced = models.JSONField(blank=True, null=True)
 
     class Meta:
         db_table = "vendors"
 
 
+class Organization(Affiliate):
+    """
+    A class used to represent an Organization that provides a service
+    associated with any of the actions.
+
+    Attributes
+    ----------
+    name: str
+      name of the Organization
+    description: str
+      description of this service
+    logo: int
+      Foreign Key to Media file represtenting the logo for this Organization
+    banner: int
+      Foreign Key to Media file represtenting the banner for this Organization
+    address: int
+      Foreign Key for Location of this Organization
+    key_contact: int
+      Foreign Key for MassEnergize User that is the key contact for this Organization
+    onboarding_date: DateTime
+      When this Organization was onboard-ed on the MassEnergize Platform for this
+        community
+    onboarding_contact:
+      Which MassEnergize Staff/User onboard-ed this Organization
+    verification_checklist:
+      contains information about some steps and checks needed for due diligence
+      to be done on this Organization eg. Organization MOU, Reesearch
+    is_verified: boolean
+      When the checklist items are all done and verified then set this as True
+      to confirm this Organization
+    more_info: JSON
+      any another dynamic information we would like to store about this Service
+    created_at: DateTime
+      The date and time that this Organization was added
+    created_at: DateTime
+      The date and time of the last time any updates were made to the information
+      about this Organization
+    is_approved: boolean
+      after the community admin reviews this, can check the box
+    
+    """
+
```

Comment:
Add a field allowed_users : a ManyToMany to UserProfile, which would be all users permitted to post events on behalf of the organization.
Initially set to the user who created the organization, but then can add other users who post events for the organization (after checking with the  primary user that this user can post.
Community Admins can always post for organizations in their community.  Super admins can post on behalf of any orgqnization.